source/production/F0.Minesweeper.App/F0.Minesweeper.App.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
+
+	<PropertyGroup>
+		<TargetFramework>net5.0</TargetFramework>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="5.0.0" />
+		<PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="5.0.0" PrivateAssets="all" />
```

Comment:
Do we want to update to the latest stable release versions `5.0.2`?

Reply:
I made it `5.0.3` - and also updated the .NET SDK (`global.json`)

Reply:
also added explicit [PreferExactMatches](https://docs.microsoft.com/en-us/aspnet/core/migration/31-to-50?tabs=visual-studio#changes-to-blazor-app-routing-logic-in-501-and-further-5x-releases-up-to-60)